assets/scss/buttons-links.scss

```diff
@@ -0,0 +1,95 @@
+.btn-primary {
+  background: $blue;
+  border-color: $blue;
+}
+
+.btn-link {
+  color: $blue;
+}
+
+.btn-outline-primary {
+  border-color: $blue;
+  color: $blue;
+  font-size: 0.875rem;
+  text-transform: uppercase;
+
+  &:hover {
+    background-color: $blue;
+    border-color: inherit;
+  }
+}
+
+a {
+  &.skip-main {
+    left: -999px;
+    position: absolute;
+    top: auto;
+    width: 1px;
+    height: 1px;
+    overflow: hidden;
+    z-index: -999;
+  }
+
+  &.skip-main:focus, &.skip-main:active {
+    color: #fff;
+    background-color: #000;
```

Comment:
Is there a variable for black?

Reply:
and white?

Reply:
Ah, yes, that is the skip to main accessibility link. It now uses variables.